super-mode-calculator/cfn-dynamodb.yaml

```diff
@@ -0,0 +1,47 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: Super mode dynamodb table
+
+Parameters:
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - CODE
+      - PROD
+
+Resources:
+  SuperMode:
+    Type: AWS::DynamoDB::Table
+    Properties:
```

Comment:
Does the table need a retention policy?

Reply:
maybe later. This has the record of all super mode articles so it's useful to have here for now

Reply:
Yeah for now this is our complete history of super mode. Perhaps in the future it would make sense to add a retention policy and sync this data into datalake for recording the history of super mode?